### ISSUE TITLE
ACCESS-CM2-tasmax.yaml fails cleaning validation with nans in ssp585 and ssp126

### ISSUE THREAD

**Issue by brews** [ClimateImpactLab/downscaleCMIP6#367]
tasmax end-to-end run for ACCESS-CM2 fails in the cleaning cmip6-validation step. Logs report nans in ssp585 and ssp126.

ssp585 logged error:
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/CSIRO-ARCCSS/ACCESS-CM2/ssp585/r1i1p1f1/day/tasmax/gn/v20210317.zarr
/opt/dodola/dodola/core.py:652: UserWarning: projection cmip6 file has excess timesteps, has 39055
  warnings.warn(
Traceback (most recent call last):
  File "/argo/staging/script", line 44, in <module>
    tasks = dask.compute(*tasks)
  File "/opt/conda/lib/python3.9/site-packages/dask/base.py", line 570, in compute
    results = schedule(dsk, keys, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/dask/threaded.py", line 79, in get
    results = get_async(
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 507, in get_async
    raise_exception(exc, tb)
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 315, in reraise
    raise exc
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 220, in execute_task
    result = _execute_task(task, data)
  File "/opt/conda/lib/python3.9/site-packages/dask/core.py", line 119, in _execute_task
    return func(*(_execute_task(a, cache) for a in args))
  File "/argo/staging/script", line 26, in clear_memory_intensive_tests
    _test_for_nans(d, v)
  File "/opt/dodola/dodola/core.py", line 634, in _test_for_nans
    assert ds[var].isnull().sum() == 0, "there are nans!"
AssertionError: there are nans!
```

log from failed ssp126:
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/CSIRO-ARCCSS/ACCESS-CM2/ssp126/r1i1p1f1/day/tasmax/gn/v20210317.zarr
/opt/dodola/dodola/core.py:652: UserWarning: projection cmip6 file has excess timesteps, has 39055
  warnings.warn(
Traceback (most recent call last):
  File "/argo/staging/script", line 44, in <module>
    tasks = dask.compute(*tasks)
  File "/opt/conda/lib/python3.9/site-packages/dask/base.py", line 570, in compute
    results = schedule(dsk, keys, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/dask/threaded.py", line 79, in get
    results = get_async(
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 507, in get_async
    raise_exception(exc, tb)
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 315, in reraise
    raise exc
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 220, in execute_task
    result = _execute_task(task, data)
  File "/opt/conda/lib/python3.9/site-packages/dask/core.py", line 119, in _execute_task
    return func(*(_execute_task(a, cache) for a in args))
  File "/argo/staging/script", line 26, in clear_memory_intensive_tests
    _test_for_nans(d, v)
  File "/opt/dodola/dodola/core.py", line 634, in _test_for_nans
    assert ds[var].isnull().sum() == 0, "there are nans!"
AssertionError: there are nans!
```

This is from workflow https://argo.cildc6.org/workflows/default/e2e-access-cm2-tasmax-nnml6

Blocking progress on run #211.

**Comment by brews** [comment]
Heads up to @dgergel

**Comment by dgergel** [comment]
@brews thanks. I looked into this and both SSPs 126 and 585 have NaNs for all gridcells for the entire future projection period. Recommend eliminating SSPs 126 and 585 for ACCESS-CM2, and I'll create two related issues: 1) updating the parameter files, 2) finding out if there is an updated version in Pangeo and if not, tracking down why these SSPs entirely have NaNs with ESGF and/or CSIRO-ARCCSS. We can confine our first round deliverable for this model to SSPs 245 and 370.

**Comment by dgergel** [comment]
updated and closed via #372 and #373
